packages/esm-patient-chart-app/src/patient-banner-tags/visit-attribute-tags.extension.tsx

```diff
@@ -26,13 +26,21 @@ const getAttributeValue = (attributeType, value) => {
   }
 };
 
+/**
+ * This extension slots to the patient-banner-tags-slot by default.
+ * TODO: This component is used outside the patient chart, but incorrectly
```

Comment:
What should we do instead in those instances? Would we rely on injecting the visit by passing it down via props/state from the parent banner context and optionally fall back to fetching visit data by calling `useVisit` with the supplied `patientUuid` if it's not provided?